### ISSUE TITLE
question about training synthesis data

### ISSUE THREAD

**Issue by szlbiubiubiu** [mk-minchul/dcface#20]
Hi, thank you for your excellent work.

I have a problem when trying to train the synthesis data you released. My training would converge in one epoch and always get 50% verification accuracy on validation sets, regardless of which loss function I used. But when I switch back to casia-webface, the results would be as normal(i.e. 90+ accuracy).

Could you please let me know if there is any difference between training synthesis data and real data?

The configuration is as follows:
Start LR:0.1
weight decay: 5e-4
optimizer: SGD
network: IResnet50
Loss: Cosface(s=64, m=0.4); Arcface(s=64,m=0.5), AdaFace(from https://github.com/mk-minchul/AdaFace)

**Comment by szlbiubiubiu** [comment]
Training works after I extract the images to image format(jpg/png). Sorry for the issue.
